code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -64,10 +64,8 @@
 //all air alarms in area are connected via magic
 /area
 	var/obj/machinery/alarm/master_air_alarm
-	var/list/air_vent_names = list()
-	var/list/air_scrub_names = list()
-	var/list/air_vent_info = list()
-	var/list/air_scrub_info = list()
+	var/list/obj/machinery/atmospherics/unary/vent_pump/vents = list()
+	var/list/obj/machinery/atmospherics/unary/vent_scrubber/scrubbers = list()
```

Comment:
I hate typecasted lists because they rarely have functionality, but I do suppose there's precedent if you intend it to only have one object type.